app/mm-redux/actions/posts.ts

```diff
@@ -267,9 +270,10 @@ export function createPost(post: Post, files: any[] = []) {
             }
 
             dispatch(batchActions(actions, 'BATCH_CREATE_POST_FAILED'));
+            failed = true;
         }
 
-        return {data: true};
+        return {data: true, failed};
```

Comment:
```suggestion
        return {data: failed};
```

Reply:
you can always return early as in `return {data: true}` on line 273 and `return {data: false}` on 251 if you want, 0/5 with this one

Reply:
@enahum implemented this